cnd/src/http_api/action.rs

```diff
@@ -54,17 +54,17 @@ pub enum ActionResponseBody {
         min_median_block_time: Option<Timestamp>,
     },
     EthereumDeployContract {
-        data: crate::ethereum::Bytes,
+        data: comit::ethereum::Bytes,
```

Comment:
Can we have these all changed to just `ethereum::Bytes` please?  Can be done as follow up PR though (perhaps we should have a ticket to audit the whole codebase for unnecessary long paths).

Reply:
I don't think having a ticket helps here. It is probably not worth the time talking about but it.

If you feel strongly enough, just go ahead and change it :)
I wanted to get this over the finish line so I didn't bother changing small things like this.